eml/App.js

```diff
@@ -198,9 +188,18 @@ export default function App() {
                 options={{ headerShown: false }}
               />
               <Stack.Screen
-                name="Section"
+                name={"CourseStack"}
+                component={CourseStack}
+                options={{ headerShown: false }}
+              />
+              <Stack.Screen
+                name={"Section"}
                 component={SectionScreen}
-                initialParams={{ course_id: '' }}
```

Comment:
reason why removed this?

Reply:
shouldn't have been removed, fixed now ;))